core-plugin/src/com/google/gct/idea/debugger/ui/CloudAttachDialog.java

```diff
@@ -156,6 +159,7 @@ protected JComponent createCenterPanel() {
   @Override
   protected void doOKAction() {
     if (getOKAction().isEnabled()) {
+      doTracking();
```

Comment:
It was kind of hard to do that because everything was all over the place. I'll try again though